src/telemetry/metrics.yaml

```diff
@@ -676,6 +676,66 @@ session:
         type: boolean
 
 connection_health:
+  data_transferred_tx:
+    type: custom_distribution
+    send_in_pings:
+      - vpnsession
+      - daemonsession
+    description: |
+      The amount of data sent through the VPN tunnel.
+
+      This metric is accumulated periodically on a 3hr interval
+      while the VPN is turned on.
+
+      This metric is recorded in **bytes**. It is not collected on iOS.
```

Comment:
There is a `unit:` property that you could use for this information.  Something like `unit: bytes` would show up at the top level of the metric and be easier to use in analysis and in the tooling like Glean Dictionary.

Reply:
Thank you! Updated this for both metrics.